recipes/moltenvk/all/conandata.yml

```
@@ -36,6 +36,11 @@ sources:
     url: "https://github.com/KhronosGroup/MoltenVK/archive/v1.1.0.tar.gz"
     sha256: "0538fa1c23ddae495c7f82ccd0db90790a90b7017a258ca7575fbae8021f3058"
 patches:
+  "1.2.2":
+    - patch_file: "patches/1.2.0-0001-fix-version-number-icd-json.patch"
```

I also ran across this issue the other day, thanks for the fix!

Just a small oversight here:

```suggestion
    - patch_file: "patches/1.2.2-0001-MVKSmallVector-Make-iterator-const-correct.patch"
```

---

Thanks for the catch, that's merged.